builder/virtualbox/ovf/step_import_test.go

```diff
@@ -18,7 +18,7 @@ func TestStepImport(t *testing.T) {
 	var c Config
 	c.Prepare(cfg)
 	state.Put("vm_path", "foo")
-	state.Put("config", &c)
+	state.Put("config", c)
```

Comment:
Looking at `StepImport.Run` I don't see a need to have config within the state so you may be able to delete this line entirely.